src/trackers/classes/request.js

```diff
@@ -2,6 +2,8 @@ const shared = require('./../helpers/sharedData.js')
 const ParsedUrl = require('./../helpers/parseUrl.js')
 const getOwner = require('./../helpers/getOwner.js')
 
+const COOKIE_LENGTH_CUTOFF = 5
```

Comment:
Why did you pick 5?

Reply:
This is fairly arbitrary - the assumption is that values shorter than 5 don't have enough entropy to be of interest for tracking. We may want to come up with a better way to pick this value, but I would considered it out of scope for this PR.